crates/core/src/dom/attribute.rs

```diff
@@ -4,13 +4,36 @@ use smallstr::SmallString;
 
 use crate::InternedString;
 
+#[macro_export]
+macro_rules! attr {
+    ($name:literal) => {
+        Attribute::new($name, None)
+    };
+    ($name:literal= $value:expr) => {
+        Attribute::new($name, Some($value.to_string()))
+    };
+    ($namespace:literal : $name:literal) => {
+        Attribute {
+            name: AttributeName::new_with_namespace($namespace, $name),
+            value: None,
+        }
+    };
+    ($namespace:literal : $name:literal = $value:expr) => {
+        Attribute {
+            name: AttributeName::new_with_namespace($namespace, $name),
+            value: Some($value.to_string()),
+        }
+    };
+}
+
```

Comment:
At this moment, I think this macro is only used in tests so it should probably go behind a `#[cfg(test)]` or in a test module.